### ISSUE TITLE
Note following Heading overlap issue

### ISSUE THREAD

**Issue by dsompura** [stolksdorf/homebrewery#161]
**If you are suggestting a feature or improvement**, delete everything here and just write your idea.

If you are reporting a bug, please fill out the details. If you don't I will most likely just close the issue asking for more detail.

---

**Browser Type/Version**: Google Chrome/52.0.2743.82 m
**Operating System**: Windows 10

**Issue Description**: Any headings that has more than a single '#' followed by a note causes the note to overlap with the heading title. 

**Brew code to reproduce**:

```
## The Cottage

> You creep silently through the underbrush towards the cottage. From the doorstep, you see the man sitting inside, turned away from you. He is sharpening an axe.

### The Cottage

> You creep silently through the underbrush towards the cottage. From the doorstep, you see the man sitting inside, turned away from you. He is sharpening an axe.

#### The Cottage

> You creep silently through the underbrush towards the cottage. From the doorstep, you see the man sitting inside, turned away from you. He is sharpening an axe.
```

**Related Images** : 
![image](https://cloud.githubusercontent.com/assets/5605859/17278766/5f9e73f4-5719-11e6-8483-f01ddfac3c03.png)

![image](https://cloud.githubusercontent.com/assets/5605859/17278721/677081e0-5718-11e6-8dc4-c6afb942381c.png)

![image](https://cloud.githubusercontent.com/assets/5605859/17278765/5276f4bc-5719-11e6-899a-e47e7d234d0e.png)

**Comment by com-charizard** [comment]
I usually but a br to fix this

````
### A
<br>
> Nothe

````

**Comment by stolksdorf** [comment]
This is fixed in v2.3.0
